contracts/Identity.sol

```diff
@@ -72,29 +72,4 @@ contract Identity is KeyManager {
     // solium-disable-next-line security/no-call-value
     return to.call.value(value)(data);
   }
-
-  /**
-   * @dev Checks the purpose of keys used for signing.
-   * @param message bytes32 message to be verified. Must be generated with abi.encodePacked(arg1, arg2, arg3)
-   * @param signature bytes Signed data
-   * @param purpose uint256 of the key
-   */
-  function isSignedWithPurpose(
```

Comment:
Why did you remove the method?

Reply:
In most cases you need the ecrecover value(Ex: In Po you build the property for precise proofs with it. Also relying on a external contract to validate a signature I think it is error prone and should validate it in the contract where you need it. The identity contract just tells you if it has a key with a specific purpose and if it's revoked. Yea you might have some code duplication but that is not important IMO. 
Also the direction of the Identity standard. It's just a proxy with an execute method